docs/src/pages/components/data-grid/rendering/AutoHeightGrid.js

```diff
@@ -12,6 +12,7 @@ export default function AutoHeightGrid() {
   return (
     <div style={{ height: 400, width: '100%' }}>
       <DataGrid autoHeight {...data} />
+      <p>more content</p>
```

Comment:
Not sure we need that in the doc

Reply:
Before the fix for autoHeight, the `<p>` was overlapping with the data grid. This diff is mean for developers to quickly visualize what auto height mean.